docs/_config.yml

```diff
@@ -525,3 +525,14 @@ fr:
         - path: fr/week12/12-1.md
         - path: fr/week12/12-2.md
         - path: fr/week12/12-3.md
+
+
+#################################### Russian ####################################
+ru:
+  title: 'Глубокое обучение'
+  chapters:
+    - path: ru/week01/01.md
+      sections:
+        - path: ru/week01/01-1.md
+        - path: ru/week01/01-2.md
+        - path: ru/week01/01-3.md
```

Comment:
```suggestion
```
There is no `ru/week01/01-2.md` or `ru/week01/01-3.md` in the current tree nor in this PR. Let's add those lines when we have a PR for them.

Reply:
Lines have added